packages/backend-tasks/src/index.ts

```diff
@@ -17,7 +17,11 @@
 /**
  * Common distributed task management library for Backstage backends
  *
+ * @remarks
+ * This package is deprecated and will be removed in a near future.
+ * Please migrate to the new backend system, and depend on `coreServices.scheduler` from  `@backstage/backend-plugin-api` instead, or use `DefaultSchedulerService` from `@backstage/backend-defaults`.
+ *
  * @packageDocumentation
  */
 
-export * from './deprecated';
+export default 'This package is deprecated and will be removed in a near future. Please migrate to the new backend system, and depend on `coreServices.scheduler` from  `@backstage/backend-plugin-api` instead, or use `DefaultSchedulerService` from `@backstage/backend-defaults`.';
```

Comment:
@Rugvip If I didn't export something, the API report generation fails. Is it expected?

Reply:
Oh but hold on, we don't want to delete the contents of the package, just update the README + package.json 😅. If we remove the implementation we'll break the package for existing users.

Reply:
Just to make sure I understand, we can't delete this until we completely end support for the legacy system, right? I'll add the deleted `backend-tasks` files back.

Reply:
Files added back and package.json is marked as deprecated 🙏🏻